presentations/en/SLF.md

```diff
@@ -0,0 +1,75 @@
+# Senior Leadership Forum
+
+ESDC IT Strategy (30min presentation)
+
+## Agenda
+
+1. (5 min) Approach to IT Strategy
+2. (5 min) Team's focus areas
+3. (5 min) Recommended immediate actions
+
+(15 min) Q&A and buffer time
+
+---
+
+## 1. IT Strategy approach
+
+Purpose: Set the stage by clearly communicate that we are not doing conventional IT strategy with some examples of what that means
+
+Talk about:
+
+- Genesis of IT strategy and reason why the team got created (Dec 2018 senior leader meeting, goal of getting an integrated business-IT strategy)
+- How we are **NOT** doing this
+- We instead will focus on IT's ability to quickly respond to business changes
+- Talk about premise: the relationship between business and IT MUST change to a shared accountability one as the conventional contracting model (requirements based, a company within a company) of business-IT does not work and must move to a partnership one (outcomes based, one team). The future is too unpredictable and complex so the conventional long term planning do not work, we must instead reward an empirical cycle of trying, observing, and correcting.
```

Comment:
I'd probably define what "do not work" means to avoid having anyone take this as "we're not doing our jobs".

I don't think that's what we mean here but rather that in the Government, including ESDC, layers upon layers of governance with the silo'd approach has been used to try to manage risk but it has lead to the opposite effect.